docs-src/python/how-to-set-a-schedule-in-python.md

```diff
@@ -0,0 +1,42 @@
+---
+id: how-to-set-a-schedule-in-python
+title: How to set a Schedule in Python
+sidebar_label: Schedule
+description: Set a Schedule in Python for scheduling Workflows to be run at specific times in the future.
+tags:
+  - python
+  - developer-guide
+  - how-to
+---
+
+Use the [create_schedule()](https://python.temporal.io/temporalio.client.Client.html#create_schedule) method available in the [Client](https://python.temporal.io/temporalio.client.Client.html) class to create a new Schedule for a Workflow Execution
+
+Define the parameters of the Schedule, then call the `create_schedule()` method on the Client object and pass in the Workflow Id for the Workflow you want to schedule, as well as the Schedule object you created.
```

Comment:
Where's the action in this sample?
And note that both the workflow and schedule have IDs.